src/PrestaShopBundle/Form/Admin/Sell/Supplier/SupplierType.php

```diff
@@ -150,26 +190,28 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('phone', TextType::class, [
-                'empty_data' => '',
+                'label' => $this->trans('Phone', 'Admin.Global'),
+                'help' => $this->trans('Phone number for this supplier', 'Admin.Catalog.Help'),
                 'required' => false,
                 'constraints' => $this->getPhoneCommonConstraints(),
             ])
             ->add('mobile_phone', TextType::class, [
-                'empty_data' => '',
+                'label' => $this->trans('Mobile phone', 'Admin.Global'),
+                'help' => $this->trans('Mobile phone number for this supplier.', 'Admin.Catalog.Help'),
                 'required' => false,
                 'constraints' => $this->getPhoneCommonConstraints(),
             ])
             ->add('address', TextType::class, [
-                'empty_data' => '',
+                'label' => $this->trans('Address', 'Admin.Global'),
                 'constraints' => $this->getAddressCommonConstraints(),
             ])
             ->add('address2', TextType::class, [
-                'empty_data' => '',
+                'label' => $this->trans('Address (2)', 'Admin.Global'),
                 'required' => false,
                 'constraints' => $this->getAddressCommonConstraints(),
             ])
             ->add('post_code', TextType::class, [
-                'empty_data' => '',
+                'label' => $this->trans('Zip/postal code', 'Admin.Global'),
```

Comment:
@Junebyun, I can see this string also exists with a capital letter (_Zip/Postal code_): which one do you recommend?

Reply:
I'll go with the capital! It would be consistent with the capital letter of Zip! Thanks. :)

Reply:
It suits me fine! I let you add this decision to the wording guidelines [document](https://docs.google.com/document/d/1pI5fsmQO9Zp2PSDRicnFHuBGV1j2xpG4h5z9K4Bl4Bg/edit), thank you.